doc/CMakeLists.txt

```diff
@@ -153,14 +153,48 @@ if(${DOXYGEN_FOUND})
             # This means the user wants to use MathJax *and* we were able to
             # download it successfully.
 
-            set(mathjax_unzipped_dir "${html_binary_dir}/MathJax-2.5-latest")
+            set(mathjax_unzipped_dir "${html_binary_dir}/MathJax-2.7.0")
 
-            # This creates a folder html_${audience}/MathJax-2.5-latest in the
+            # This creates a folder html_${audience}/MathJax-2.7-latest in the
             # build tree.
             add_custom_command(
                 OUTPUT "${mathjax_unzipped_dir}"
                 COMMENT "Unzipping MathJax.zip for ${audience} documentation"
                 COMMAND ${CMAKE_COMMAND} -E tar -x "${MATHJAX_ZIP_PATH}" 
+                # Trim away unnecessary files; 170 MB -> under 5 MB.
```

Comment:
The PR comment says "Without this PR, the html_developer dir is about 150MB; after this PR, it's only about 105MB." It sounds like you have reduced the size by 45 Mb, which would suggest "170 Mb -> 125 Mb" on L164. Why only "5 MB"?